### ISSUE TITLE
Cannot get maestral started

### ISSUE THREAD

**Issue by coxackie** [SamSchott/maestral-dropbox#80]
**Describe the bug**
Cannot get `maestral` started. Note that I have installed and used it in the past, but this time it does not appear to be working.

**To Reproduce**
Installed `maestral`, per the instructions, both 0.5 and 0.5-dev, running `maestral start` or `maestral gui` leads to the same error (Traceback below).

**Expected behaviour**
Maestral should start....

**System (please complete the following information):**
 - Maestral version: 0.5 / 0.5-dev
 - OS: Arch Linux
 - Desktop environment: Gnome 3.34
 - Qt version: 5.14

**Additional context**
This is the traceback with 0.5 - the same happens with 0.5-dev.

```
Traceback (most recent call last):
  File "/usr/bin/maestral", line 11, in <module>
    load_entry_point('maestral==0.5.0', 'console_scripts', 'maestral')()
  File "/usr/lib/python3.8/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.8/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/maestral/cli.py", line 245, in gui
    from maestral.gui.main import run
  File "/usr/lib/python3.8/site-packages/maestral/gui/main.py", line 30, in <module>
    from maestral.sync.daemon import (
  File "/usr/lib/python3.8/site-packages/maestral/sync/daemon.py", line 17, in <module>
    from Pyro5 import server, client
  File "/usr/lib/python3.8/site-packages/Pyro5/server.py", line 18, in <module>
    from . import config, core, errors, serializers, socketutil, protocol, client
  File "/usr/lib/python3.8/site-packages/Pyro5/core.py", line 14, in <module>
    from . import config, errors, socketutil, serializers, nameserver
  File "/usr/lib/python3.8/site-packages/Pyro5/nameserver.py", line 269, in <module>
    @server.expose
AttributeError: partially initialized module 'Pyro5.server' has no attribute 'expose' (most likely due to a circular import)
```

**Comment by SamSchott** [comment]
@coxackie, this error comes from a slight change in imports in Pyro5 5.7 which was released very recently). I have adapted the Pyro5 imports in the develop branch and pushed a new pre-release to PyPI (v0.5.1-dev4).

This branch is very stable, so a full release should follow soon.

**Comment by coxackie** [comment]
@SamSchott, can confirm it works now, many thanks for this!

**Comment by SamSchott** [comment]
fixed in v0.5.2
